firstvoices/backend/search/signals/media_signals.py

```diff
@@ -37,6 +37,4 @@ def request_update_media_index(sender, instance, **kwargs):
 @receiver(post_delete, sender=Image)
 @receiver(post_delete, sender=Video)
 def request_delete_from_index(sender, instance, **kwargs):
-    transaction.on_commit(
-        delete_from_index.apply_async((instance.id,), link_error=link_error_handler.s())
-    )
+    delete_from_index.apply_async((instance.id,), link_error=link_error_handler.s())
```

Comment:
Was waiting on the transaction a problem here?

Reply:
Yes, so we lose the instance after the transaction gets completed, and thus the instance.id cannot be passed down.
I discussed this with Bronson for confirmation, and the race condition only existed during the save action. Thus, the transaction.on_commit could be safely removed from here.

Reply:
Yes, none of the delete from index signal have/require on_commit to function properly